app/grandchallenge/participants/models.py

```diff
@@ -54,3 +56,88 @@ def delete(self, *args, **kwargs):
 
     class Meta:
         unique_together = (("challenge", "user"),)
+
+
+def string_type_schema():
+    return {"type": "string"}
+
+
+class RegistrationQuestion(UUIDModel):
+    challenge = models.ForeignKey(
+        Challenge,
+        on_delete=models.CASCADE,
+    )
+
+    question_text = models.TextField(blank=False)
+    question_help_text = models.TextField(blank=True)
+
+    schema = models.JSONField(
+        default=string_type_schema,
+        help_text="A JSON schema definition against which an answer is validated. See https://json-schema.org/."
+        "Only Draft 7, 6, 4 or 3 are supported.",
+        validators=[JSONSchemaValidator()],
+    )
+
+    required = models.BooleanField(default=True)
```

Comment:
Maybe `question_text`, `question_help_text` and `required` should have help text?